JustSaying.UnitTests/AwsTools/MessageHandling/Sns/TopicByName/WhenPublishingAsyncExceptionCanBeHandled.cs

```diff
@@ -21,11 +21,11 @@ public class WhenPublishingAsyncExceptionCanBeHandled : XAsyncBehaviourTest<SnsT
         private readonly IMessageSerialisationRegister _serialisationRegister = Substitute.For<IMessageSerialisationRegister>();
         private readonly IAmazonSimpleNotificationService _sns = Substitute.For<IAmazonSimpleNotificationService>();
         private const string TopicArn = "topicarn";
-        private readonly MessageResultLogger _messageResultLogger = r => {};
+        private readonly Action<MessageResponse> _responseLogger = r => {};
```

Comment:
Is it worth adding a static property somewhere that all the tests (or integrations) can just use as a "no-op" for this to make it easier?

```csharp
public static Action<MessageResponse> NoOp { get; } = (_) => { };
```